src/main/java/me/ryanhamshire/GPFlags/FlagManager.java

```diff
@@ -116,8 +117,8 @@ public SetFlagResult setFlag(String claimId, FlagDefinition def, boolean isActiv
             }
             internalParameters.append(arg).append(" ");
         }
-        internalParameters = new StringBuilder(internalParameters.toString().trim());
-        friendlyParameters = new StringBuilder(friendlyParameters.toString().trim());
+        internalParameters = new StringBuilder(ChatUtil.validatedHexString(sender, internalParameters.toString()).trim());
```

Comment:
When the server restarts, the plugin goes through each item in flags.yml and runs setFlag on them. When it's run on startup like this, the sender is the console -- not the player who set the flag. 

I think that actually should be fine since the first time its run, the sender is the player and we've already removed the invalid permissions at that point.

Reply:
This part is ok, because we are having two cases:
- The player sets the flag: the validation happens
- the plugin sets the flag: the sender is null and therefore returns the message directly. You can look it up under `ChatUtil.validateHexString` and I tested this on a server too. Without null check it doesnt set the flags on reload properly.